packages/agents/lighthouse/src/config.ts

```diff
@@ -160,6 +162,8 @@ export const getEnvConfig = (
       configJson.relayerWaitTime ||
       configFile.relayerWaitTime ||
       DEFAULT_RELAYER_WAIT_TIME,
+    concurrency:
+      process.env.NXTP_PROVER_CONCURRENCY || configJson.concurrency || configFile.concurrency || DEFAULT_CONCURRENCY,
```

Comment:
In general, the data type of env variable is string but concurrency is number so need a fix here

Reply:
I think this is correct.

Reply:
True. Pushed one way check and convert, perhaps there is a more elegant way to do this.

Interesting that we have this pattern at a few different places like  for `relayerWaitTime` , `NXTP_CARTOGRAPHER_POLL_INTERVAL`. 
We should update this in all places right @rhlsthrm @wanglonghong  ?

Reply:
Yes right. we have similar things in other places.

Reply:
Correct I think it's working because we are using the defaults, so we need this if we override it.